omnibus/config/projects/chef-workstation.rb

```diff
@@ -99,6 +99,11 @@
 dependency "uninstall-scripts"
 dependency "ruby-cleanup"
 
+dependency "go"
```

Comment:
you shouldn't need the go dependency here if the analyze project config also deps on it

Reply:
We discussed this and wanted to make it explicit because there's no clear indication of what's installing the go binary, or even if it's installed, at the time we add the cleanup dependency further down. It was more for humans than for omnibus.

Reply:
What Marc said ☝️ -- Though, I would like to ask @tyler-ball for his point of view. This is just a matter of clarity for the team since now we will be shipping Ruby, Rust and Go binaries.